lib/src/Uncertainty/Algorithm/Transformation/MarginalTransformation/openturns/MarginalTransformationEvaluation.hxx

```diff
@@ -60,6 +60,7 @@ public:
   /** Parameter constructor */
   MarginalTransformationEvaluation(const DistributionCollection & inputDistributionCollection,
                                    const DistributionCollection & outputDistributionCollection,
+                                   const UnsignedInteger direction = FROM,
```

Comment:
@jschueller No. FROM is here only to map a given set of distributions into Uniform(0,1) ones, so it should not be used as the defaukt value here. In fact, the attribute 'direction_' is for internal use only and its visibility should be reduced as much as possible